designs/tripleo/osc-tripleo.md

```diff
@@ -68,7 +68,7 @@ description: >
     description: image
     type: string
 	
-  # The .
+  # The source of the mounted volume OSC needs to persist information.
```

Comment:
I guess the way the param's passed to Docker run argument list is that via the "resource" section we need to have docker run section under resource creation list, and this docker run should have heat template arguments list. 
we will post this question as part of TLS update.

Reply:
I dont think there is an explicit docker run section. The creation of the container happens due to the section `docker_config`. I believe this will trigger OOO to apply/create the container, in that section you can use OOO parameters for env variables or other things like volumes, for instance here `{get_param: OSCVolumeSrc}:/opt/vmidc/bin/data/`

Reply:
Yes that is right there is no explicit section called docker run. I mean in the "resource" section we can call Docker run. I had seen links/article (i don't recollect), as part of resource section they were initiating the docker run. My question was if done like this, we can pass the parameters as part of this definition.

Reply:
Yes we should be able to.